### PR TITLE
Update swift-nio-ssl to latest HEAD.

### PR DESCRIPTION
**Pull request by Lukasa**
This resolves errors with expired certs in the tests.

### CODE REVIEW

**Comment by justice-adams-apple** [comment]
@swift-ci test

**Review by justice-adams-apple — APPROVED** [review]
LGTM!

**Comment by Lukasa** [comment]
FYI that I don't have write access here so I can't merge this patch.

**Pull request merged**
